README.md

```diff
@@ -131,7 +124,7 @@ path /elastic_search/
 scheme https
 ```
 
-You can specify user and password for HTTP basic auth. If used in conjunction with a hosts list, then these options will be used by default i.e. if you do not provide any of these options within the hosts listed.
+You can specify user and password for HTTP Basic authentication.
```

Comment:
Previous implementation assumes that users can use multiple Elasticsearch cluster in one ES plugin instance.
I reckon that we should write configuration migration guide.

Otherwise, I agree with removing basic authentication description for multiple hosts.

Reply:
In 4ae5697, I added a small note to tell users to migrate from URL-embedded user credentials.